autonomy/cli/helpers/chain.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""On-chain interaction helpers."""
+import json
+from collections import OrderedDict
+from pathlib import Path
+from typing import Any, Dict, List, Optional, cast
+
+import click
+from aea.configurations.constants import DEFAULT_README_FILE
+from aea.configurations.data_types import PackageType, PublicId
+from aea.configurations.loader import load_configuration_object
+from aea.crypto.base import Crypto, LedgerApi
+from aea.helpers.cid import CID
+from aea.helpers.ipfs.base import IPFSHashOnly
+from aea_cli_ipfs.ipfs_utils import IPFSTool
+from aea_ledger_ethereum.ethereum import EthereumApi
+
+from autonomy.chain.base import ComponentRegistry, RegistryManager
+from autonomy.chain.config import COMPONENT_REGISTRY_LOCAL, REGISTRIES_MANAGER_LOCAL
+from autonomy.configurations.base import PACKAGE_TYPE_TO_CONFIG_CLASS
+
+
+BASE16_HASH_PREFIX = "f01701220"
+CONFIG_HASH_STRING_PREFIX = "0x"
+UNIT_HASH_PREFIX = CONFIG_HASH_STRING_PREFIX + "{metadata_hash}"
+DEFAULT_NFT_IMAGE_HASH = "bafybeiggnad44tftcrenycru2qtyqnripfzitv5yume4szbkl33vfd4abm"
```

Comment:
What does this resolve to?

Reply:
To the image which jose uses for docs and demos (https://gateway.autonolas.tech/ipfs/bafybeiggnad44tftcrenycru2qtyqnripfzitv5yume4szbkl33vfd4abm)